src/hub/HubHelpers.sol

```diff
@@ -0,0 +1,183 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.28;
+
+import {D18, d18} from "src/misc/types/D18.sol";
+import {MathLib} from "src/misc/libraries/MathLib.sol";
+import {IERC7726} from "src/misc/interfaces/IERC7726.sol";
+import {Auth} from "src/misc/Auth.sol";
+
+import {ShareClassId} from "src/common/types/ShareClassId.sol";
+import {AssetId} from "src/common/types/AssetId.sol";
+import {AccountId} from "src/common/types/AccountId.sol";
+import {PoolId} from "src/common/types/PoolId.sol";
+
+import {IHubHelpers} from "src/hub/interfaces/IHubHelpers.sol";
+import {IAccounting} from "src/hub/interfaces/IAccounting.sol";
+import {IHubRegistry} from "src/hub/interfaces/IHubRegistry.sol";
+import {IShareClassManager} from "src/hub/interfaces/IShareClassManager.sol";
+import {IHoldings} from "src/hub/interfaces/IHoldings.sol";
+import {IHub, AccountType} from "src/hub/interfaces/IHub.sol";
+
+contract HubHelpers is Auth, IHubHelpers {
+    using MathLib for uint256;
+
+    IHoldings public immutable holdings;
+    IAccounting public immutable accounting;
+    IHubRegistry public immutable hubRegistry;
+    IShareClassManager public immutable shareClassManager;
```

Comment:
Maybe CR: Curious why for the helper these are set to `immutable` whereas for the hub that's not the case. Seems erroneous to me because they should in sync regarding mutability?

Reply:
Making them `immutable` in the` Hub` increases the bytecode size :(